src/compute/src/memory_management/mod.rs

```diff
@@ -122,6 +115,14 @@ impl MemoryControl for DummyPolicy {
 /// overhead, network buffer, etc. based on `SYSTEM_RESERVED_MEMORY_PROPORTION`. The reserve memory
 /// size must be larger than `MIN_SYSTEM_RESERVED_MEMORY_MB`
 pub fn reserve_memory_bytes(total_memory_bytes: usize) -> (usize, usize) {
+    if total_memory_bytes < MIN_COMPUTE_MEMORY_MB << 20 {
+        panic!(
+            "The total memory size ({}) is too small. It must be at least {} MB.",
+            convert(total_memory_bytes as _),
+            MIN_COMPUTE_MEMORY_MB
+        );
+    }
```

Comment:
Don't know why `MIN_COMPUTE_MEMORY_MB` was not checked anywhere, so I added it by the way.